<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base_converter</class>
 <widget class="QDialog" name="base_converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Convert Hex, Dec, Bin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_data">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="1,5">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="cb_frombase">
          <item>
           <property name="text">
            <string>Decimal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Binary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hexadecimal</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_to">
          <property name="text">
           <string>To</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_tobase">
          <item>
           <property name="text">
            <string>Decimal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Binary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hexadecimal</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_input">
        <property name="text">
         <string>Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>From</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_input"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_convert">
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,21">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>13</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_ans">
       <property name="text">
        <string>Answer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_ans">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
